frontend/client/src/real/java/de.amosproj3.ziofa.client/RustClient.kt

```diff
@@ -99,9 +104,15 @@ class RustClient(private val inner: uniffi.client.Client) : Client {
 
     override suspend fun getConfiguration(): Configuration = inner.getConfiguration().into()
 
-    override suspend fun setConfiguration(configuration: Configuration) =
+    override suspend fun setConfiguration(configuration: Configuration): UInt =
```

Comment:
is this a return code that signals error or success?

Reply:
Currently there is always 0 being returned. Should get refactored.

Reply:
Before the return value just wasn't used iirc, I wanted to make it clear that it's there.

Reply:
I dont quite understand the response type as well? Is there something else than error or success? If not, the request is already a result. If yes, we can use a oneof.

Reply:
No, as said, the response is static (0), has no need and deserves to get refactored away.

Reply:
Done